scene/2d/tile_map.h

```diff
@@ -186,9 +186,9 @@ class TileMap : public Node2D {
 	bool compatibility_mode = false;
 	bool centered_textures = false;
 	bool clip_uv = false;
-	float fp_adjust = 0.00001;
-	float friction = 1.0;
-	float bounce = 0.0;
+	real_t fp_adjust = 0.00001;
```

Comment:
This variable is no longer necessary after you replaced the only usage with `CMP_EPSILON`.

Although I wouldn't touch TileMap code at all, considering the coming rewrite.